oscarnnn/DAT076-Schedule
# Feature/participate

To test: 
1. Click on an event and click the "PARTICIPATE" button and look if the user is in the participants field in the database for the selected event(the user should be there).

2. Click on the "LEAVE" button when the event is selected and look if the user is in the participants field in the database for the selected event(the user should NOT be there).
